REQUIRE

```diff
@@ -1,5 +1,5 @@
 julia 0.3
-Compat 0.8.0
+Compat 0.10
```

Comment:
if you aren't using any features that weren't present in 0.8, then the bound should be left at 0.10

it depends if you're using any newer features, can you remember why you increased this?